sass/blocks/body.sass

```diff
@@ -0,0 +1,40 @@
+// Body Block
+
+.display
+  float: right
```

Comment:
Interesting how we didn't cover floats, yet you used them for this assignment. Not wrong at all! Just interesting!